change/@fluentui-web-components-2020-09-24-11-27-41-bhkotak-swatch-update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
+  "comment": "Merge branch 'master' of https://github.com/bhavinkotak07/fluentui into bhkotak/swatch-update",
```

Comment:
Delete this file, it shouldn't be needed

Reply:
Done.